src/components/ForecastChart/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import PropTypes from 'prop-types';
+
+import './styles.css';
+
+const ForecastChart = ({ forecastData }) => {
+  const dates = forecastData.map(forecast => forecast.date);
+  const temperatures = forecastData.map(forecast => forecast.temp);
+  const label = "Max temperature";
+  const backgroundColor = "rgba(174,234,0,0.4)";
```

Comment:
If this is const why this is not in styles?

Reply:
Not really sure if I get it here. This needs to be passed to chart (Line) component to style the chart, I just extracted this here for better visibility.

Reply:
Things like background color I would look in `styles.css` file not in source code. If there is condition for setting some additional styles for any component you can either pass `className` to you component as a `prop` or add some prop which represent some style like `<Popup open={isPopupOpen}>` which will add class `open` to sub-component Popup.

Passing style as a data is confusing. Later if you will need to change it you would be just confused where it is. Maybe if this comes from backend and you just must do this that way, and does styles are different for every entry but if it's same for all I see no benefits only cons

Reply:
Yep, I agree. I use external existing component here though: https://github.com/jerairrest/react-chartjs-2 which gets everything passed as data unfortunately (and it mixes stuff like labels, datasets with styling). 

So I would either need to write my own wrapper for some JS chart to avoid this. I understand the implications and I can write my own chart component but I wanted to use something existing to save some time and also integrate external components in my workflow :)

Reply:
OK, but did you tried className?

Reply:
It produces a chart in canvas element so I am not sure what would I target really.
<img width="900" alt="zrzut ekranu 2018-01-26 o 12 09 08" src="https://user-images.githubusercontent.com/6452013/35437271-ef02b102-0291-11e8-94d5-3cdf8e5018aa.png">

I think that the name of the attribute that this chart accepts just coincidentally is named the same as the css property:
http://www.chartjs.org/docs/latest/#creating-a-chart

I might be missing something, but I don't know how could I style it through regular css.

Reply:
OK, if this is working with your options then carry on. This chart is really weird :/